### PR TITLE
DGC-448 Style Program

### PR DESCRIPTION
**Pull request by charlieartist**
Since 440 failed with merge conflicts, it was rebuilt here.

### CODE REVIEW

File: docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	//border: 1px solid #f00;
+	
+}
+
+
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-sort-by,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-sort-order,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-field-experience-level-value,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-field-session-track-target-id,
+.program-schedule .views-exposed-form .form-actions.js-form-wrapper.form-wrapper {
```
Comment (lkopacz) [inline docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss]: @charlieartist I think that we can probably remove the `.form-item.js-form-type-select.form-type-select` from each line item. The specificity is already good enough without all those classes and may make it more difficult to debug in the future.

File: docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+.program-schedule .views-row:nth-child(2) {
+	background: rgba(255,0,0,1);
+}
+
+
+.program-schedule .views-row:nth-child(even) {
+	background: rgba($green, 0.1);
+}
```
Comment (lkopacz) [inline docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss]: is the `.views-row:nth-child(even)` duplicative of the `.views-row:nth-child(2)` or am I missing something?

Comment (charlieartist) [inline docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss]: @lkopacz I will take it out. As is often the case with components within View displays, there are a number of elements that need to be worked around.

File: docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss (hunk 1 above)
Comment (charlieartist) [inline docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss]: broken_tab_display
@lkopacz This is what happens when I remove those selectors. The beauty of Views!

**Comment by mikemadison13** [comment]
@charlieartist FYI all of the other "session scheduling" topics were setup with custom field permissions so they were admin only. i did this with your schedule day... however i'm a little worried that @jdearie isn't going to want to have to go back through and re-edit all of the sessions again to add the schedule day.

**Pull request closed without merging**